syncer/syncer.go

```diff
@@ -1321,6 +1321,7 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 		}
 
 		if err != nil {
+			// TODO: wrap the error with terror, and attach binlog position
```

Comment:
is still need this commnet?

Reply:
this line should be deleted 😂 We should delete it in master branch and cherry pick here (could be another PR, or left for any future PR by the way)

Reply:
https://github.com/pingcap/dm/pull/1699/commits/889a062925c9f043592457bd072798f32d5a8cdd